packages/contracts-bedrock/test/L2/SuperchainWETH.t.sol

```diff
@@ -25,18 +24,26 @@ contract SuperchainWETH_Test is CommonTest {
     /// @notice Emitted when a withdrawal is made.
     event Withdrawal(address indexed src, uint256 wad);
 
-    /// @notice Emitted when an ERC20 is sent.
-    event SendERC20(address indexed _from, address indexed _to, uint256 _amount, uint256 _chainId);
+    /// @notice Emitted when a crosschain transfer mints tokens.
+    event CrosschainMinted(address indexed to, uint256 amount);
```

Comment:
This doesn't follow the spec: https://github.com/ethereum-optimism/specs/blob/main/specs/interop/token-bridging.md#crosschainmint-1

It is still in past tense

Reply:
We can do this in a follow up

Reply:
Yes! The PR changing the tense is ready, we were just waiting this one to be merged